site/docs/glossary/types.md

```diff
@@ -24,7 +24,7 @@ An [event](https://github.com/ethereum-optimism/optimism/blob/develop/packages/c
 
 ## WithdrawTransaction
 
-A withdraw transaction is a special type of transaction format specific to the OP Stack. It is for creating L1 transactions from L2. The withdrawal flow consists of three parts. First, there is the withdraw initiating transaction which the user submits on L2 and emits a [MessagePassedEvent](#messagepassedevent). Second, there is the withdrawal proving transaction which is submitted on L1 by the user to prove legitimacy of the withdrawal. This transaction emits a [WithdrawalProvenEvent](#withdrawalprovenevent). Lastly, there is the withdrawal finalizing transaction which is submitted on L1 by the user after the fault challenge period has passed which emits a [WithdrawalFinalizedEvent](#withdrawalfinalizedevent) and executes the transaction on L1.
+A withdraw transaction is a special type of transaction specific to the OP Stack. It is for creating L1 transactions from L2. The withdrawal flow consists of three parts. First, there is the withdraw initiating transaction which the user submits on L2 and emits a [MessagePassedEvent](#messagepassedevent). Second, there is the withdrawal proving transaction which is submitted on L1 by the user to prove legitimacy of the withdrawal. This transaction emits a [WithdrawalProvenEvent](#withdrawalprovenevent). Lastly, there is the withdrawal finalizing transaction which is submitted on L1 by the user after the fault challenge period has passed which emits a [WithdrawalFinalizedEvent](#withdrawalfinalizedevent) and executes the transaction on L1.
```

Comment:
@roninjin10 I think this is confusing because it's actually not a transaction type like DepositTx is, right? There are no transactions created from deposits. There is just outputs and things proven about them. Currently this mirrors the DepositTransaction language. I think maybe we just say "An L2 to L1 transaction" and leave the details for other sections. Or if not I think worth being clear that this is creating L2 state that can later be proved on L1 and allow an L1 tx to be triggered.

Reply:
Makes sense. You should suggest a rewording if you got an idea of how to make it better